components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/ActionManagementServiceImpl.java

```diff
@@ -78,8 +80,10 @@ public Action addAction(String actionType, Action action, String tenantDomain) t
         validateMaxActionsPerType(resolvedActionType, tenantDomain);
         doPreAddActionValidations(action);
         String generatedActionId = UUID.randomUUID().toString();
-        return CACHE_BACKED_DAO.addAction(resolvedActionType, generatedActionId, action,
+        Action createdAction = CACHE_BACKED_DAO.addAction(resolvedActionType, generatedActionId, action,
                 IdentityTenantUtil.getTenantId(tenantDomain));
+        auditLogger.printActionAuditLog(ActionManagementAuditLoggerUtil.Operation.ADD, createdAction);
```

Comment:
shall we just say "printAuditLog"

Reply:
This is updated with c81491d0d97828a7760ca1c4dc028f0ab9cd0b8b